PROTOCOL.md

```diff
@@ -260,6 +260,12 @@ etc). `NAME` is the nickname of the Pokemon performing the action.
 > `switch` means it was intentional, while `drag` means it was unintentional
 > (forced by Whirlwind, Roar, etc).
 
+`|swap|POKEMON|POSITION|CAUSE`
+
+> Moves already active `POKEMON` to active field `POSITION` where `0` is the
+> leftmost position and each position to the right increments by 1. `CAUSE`
+> is optional and only used for debug.
```

Comment:
Oh I didn't see that because it was in the `Minor actions` section. This is a "Major action" correct? Should I just not list `CAUSE` at all because it doesn't look like any of the minor actions do?